examples/with_winit/src/lib.rs

```diff
@@ -701,6 +770,8 @@ fn run(
     let debug = vello::DebugLayers::none();
 
     let mut app = VelloApp {
+        present_id: 0,
+        google_display_timing_ext_devices: vec![None; renderers.len()],
```

Comment:
Should keep the same order as the declaration and put them in the opposite order under `renderers` ?